exp/controllers/azuremachinepool_controller.go

```diff
@@ -43,7 +45,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
-	capzcntr "sigs.k8s.io/cluster-api-provider-azure/controllers"
+	controllers "sigs.k8s.io/cluster-api-provider-azure/controllers"
 	"sigs.k8s.io/cluster-api-provider-azure/util/reconciler"
 
 	infrav1 "sigs.k8s.io/cluster-api-provider-azure/api/v1alpha3"
```

Comment:
nit: imports should likely be re-grouped and re-sorted

Reply:
Resorted. TY